src/main/java/org/ice4j/ice/harvest/MappingCandidateHarvesters.java

```diff
@@ -319,6 +319,8 @@ public StunMappingCandidateHarvester call()
             }
         }
 
+        es.shutdown();
```

Comment:
Can we also name the threads? I've been wondering what "pool-1-thread-1" is for a long time.